Karel needs this for his scripts that manages teams in HC.

--- /dev/null
+++ healthchecks-33d8a1505c4fee179c92ee0b9f34ff7510669f78/hc/api/migrations/0041_auto_20180711_1752.py
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0040_auto_20180517_1336'),
+    ]
+
+    operations = [
+        migrations.RunSQL('''
+            ALTER TABLE accounts_member
+            ADD CONSTRAINT accounts_member_team_id_user_id_key
+            UNIQUE (team_id, user_id)
+        ''')
+    ]
